src/Raven.Studio/typescript/viewmodels/database/indexes/additionalSourceSyntax.ts

```diff
@@ -43,18 +62,36 @@ map('People', function (person) { return {
     };
 })`;
 
-    additionalSourceCsharpHtml = ko.pureComputed(() => {
-        return Prism.highlight(additionalSourceSyntax.additionalSourceCsharp, (Prism.languages as any).csharp);
-    });
-    usageInMapCsharpHtml = ko.pureComputed(() => {
-        return Prism.highlight(additionalSourceSyntax.usageInMapCsharp, (Prism.languages as any).csharp);
-    });
-    additionalSourceJavascriptHtml = ko.pureComputed(() => {
-        return Prism.highlight(additionalSourceSyntax.additionalSourceJavascript, (Prism.languages as any).javascript);
-    });
-    usageInMapJavascriptHtml = ko.pureComputed(() => {
-        return Prism.highlight(additionalSourceSyntax.usageInMapJavascript, (Prism.languages as any).javascript);
-    });
-    
+    static readonly additionalSourceCsharpHtml = Prism.highlight(additionalSourceSyntax.additionalSourceCsharpText, (Prism.languages as any).csharp);
```

Comment:
why do we assign this to variable? Can;t we just call Prism.highlight when creating samples?

Reply:
right... done